webapp/app.py

```diff
@@ -113,28 +130,39 @@ def logout():
     """
     Empty the session, used to logout.
     """
-    flask.session.pop("openid", None)
+    flask.session.pop("authentication_token", None)
+    flask.session.pop("macaroon_root", None)
+    flask.session.pop("macaroon_discharge", None)
```

Comment:
This one is no longer stored on the session at any point is it?

Reply:
So my question is: can we just `session.clear()` ? Is there anything in the session we want to keep?

Reply:
Our code here shouldn't claim a monopoly on the flask session. We could feasibly want to do many other things with flask.session in other parts of the site that are nothing to do with login. So we shouldn't unilaterally clear the session.